### PR TITLE
JNG-5299 Model Performance Issue

### PR DESCRIPTION
**Pull request by robertcsakany**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-5299" title="JNG-5299" target="_blank">TaskJNG-5299</a>  Improve model transformation performance
  </td></table>
  <br />

### CODE REVIEW

File: judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl [hunk 1]
```diff
@@ -19,5 +19,5 @@ post {
         }
     }
 
-    asmUtils.validateUniqueXmiids();
+    // asmUtils.validateUniqueXmiids();
```
Comment (gaborflorian) [inline judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl:22]: Will this commented-out line be used later? If not, we should remove it.

File: judo-tatami-psm2asm/src/main/epsilon/transformations/asm/utils/id.eol [hunk 2]
```diff
@@ -1,52 +1,7 @@
...
-
-operation ASM!EStringToStringMapEntry getId() : String {
-     return self.eResource.getId(self);
-}
-
-// EStringToStringMapEntry
-/////////////////////////////////////////
+operation Any getId() : String {
+    return self.eResource.getId(self);
+}
```
Comment (gaborflorian) [inline judo-tatami-psm2asm/src/main/epsilon/transformations/asm/utils/id.eol:7]: Add new empty line

File: judo-tatami-psm2measure/src/main/epsilon/transformations/measure/psmToMeasure.etl [hunk 3]
```diff
@@ -2,5 +2,5 @@ import "modules/measure.etl";
 import "modules/unit.etl";
 
 post {
-    measureUtils.validateUniqueXmiids();
+    // measureUtils.validateUniqueXmiids();
```
Comment (gaborflorian) [inline judo-tatami-psm2measure/src/main/epsilon/transformations/measure/psmToMeasure.etl:5]: Will this commented-out line be used later? If not, we should remove it.

File: judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl (hunk 1 above)
Comment (bencegelei) [inline judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl:22]: I suggest creating a ticket for validating unique xmiid-s only if it's required by setting a build parameter.

**Review by bencegelei — CHANGES_REQUESTED** [review]

File: judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl (hunk 1 above)
Comment (bencegelei) [reply judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl:22]: nvm

**Review by bencegelei — APPROVED** [review]

File: judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl (hunk 1 above)
Comment (robertcsakany) [reply judo-tatami-psm2asm/src/main/epsilon/transformations/asm/psmToAsm.etl:22]: No, it can be used when developing transformations.

**Review by noherczeg — APPROVED** [review]

**Pull request merged**